docs/DeveloperGuide.adoc

```diff
@@ -279,104 +277,440 @@ Refer to the guide <<DevOps#, here>>.
+
+|`* *`|student|view unfinished tasks|I can have a easy overview of what I need to to next|Shui Yao
+
+|`*`|student|view overdue tasks|I can re-schedule my revision and be more careful with the workload in my next planning|Shui Yao
+
+|`* *`|student|get notified when a revision task is about to start|I can finish up whatever stuff at hand and get ready for revision|Shui Yao
+
+|`* *`|student|disable notification|I will not be bothered in some important events|Shui Yao
+
+|`* * *`|student|change the difficulty level of questions|I can customize the difficulty level of the questions as I progress along the time|Xueting
+
+|`* * *`|student|find all the questions at a particular difficulty level|I can see how well I understand the subject content|Xueting
+
+|`* *`|student|set a mix of different difficulty levels at exam mode|I can better simulate the real-time exams conditions|Xueting
+
+|`* *`|student|get a statistics of the number of questions I have attempted at a difficulty level|I can see how often I have practiced for this subject|Xueting
+
+|`* *`|student|get statistics of the accuracy level of the specific difficulty|I can visualise how well I have practiced for this subject|Xueting
+
+|`*`|student|get statistics of the change of difficulty levels I have made to a question|I can understand my improvement in understanding the content|Xueting
+
+|`* * *`|student|see how well I am able to answer questions for different topics/subjects|focus more on topics/subjects that I need to improve on|Irene
+
+|`* * *`|student|check my past responses to different questions|compare with my current response and improve on it if necessary/check if I am making the same mistakes twice|Irene
+
+|`* *`|student|have a graphical report of my progress|get an overview of my progress with a glimpse|Irene
+
+|`* *`|student|check the questions that I have attempted for a specific time period|I can skip questions that I have already done during that time period|Irene
+
+|`*`|student|get an overview to see the number of notes added for each topic/subject|I can see which topic/subject I need to make more notes on|Irene
+
+|`* * *`|student|mark and store the questions I have made mistakes on|I can get a better understanding of which part I need to put more time to study|Dongjun
 
-_{More to be added}_
+|`* * *`|student|take a series questions as a mock test before exam|I can refresh my memory on different modules before exam|Dongjun
+
+|`* *`|student|find specific questions according to the keywords|I can filter the questions I want for different purposes|Dongjun
+
+|`* *`|student|change the difficulty of my test|I can be better prepared for actual exam|Dongjun
+
+|`*`|student|view all questions and answers stored|I can refer them as revision guide according to different subjects|Dongjun
+
+|=======================================================================
```

Comment:
Maybe you can consider ranking the user stories by priority to make it more organised?
You can consider adding user stories for fundamental features like add/edit/delete etc for users other than students.